parser.js

```diff
@@ -0,0 +1,113 @@
+import { RootNode, Node, TextNode } from './class.js';
+
+const removeLineBreaks = (str) => str.replace(/(\r\n|\n|\r)/gm, '');
+
+/* @params attributesToBeParsed : ['class="container"'] */
+/* @return attributes : [['class', 'container']] */
+const parseAttributes = (attributesToBeParsed) => {
+  return attributesToBeParsed.map((v) => {
```

Comment:
```suggestion
const parseAttributes = (attributesToBeParsed) => attributesToBeParsed.map((v) => {
```

다른 화살표 함수와 같이 암시적 반환을 사용하는건 어떨까요? 

아니면 한 줄로 작성할 수 있는 화살표 함수만 암시적 반환을 한다 같은 기준이 있으신걸까요~?

Reply:
저도 대부분의 경우에 암시적 반환으로 표현하는 것을 선호하는데, 
예상해주신 것처럼 `Array.prototype.map`과 같은 함수에 넘겨주는 콜백이 라인을 두 줄 이상으로 넘어가면 
개인적으로 화살표의 끝이 갈 곳을 잃은 것 같고(?) 반환값이 무엇인지 조금 눈에 조금 잘 안들어오는 것 같습니다,, 😅

단지 제게 익숙하지 않아서 일수 있겠네요... 
그래도 요정도는 취향의 영역으로 볼 수 있을 것 같아 요 부분은 유지해보도록 하겠습니다,,!

 <img width="500" src="https://user-images.githubusercontent.com/60066472/137135819-94475894-8b06-457c-9700-4734eb9ac173.png">

Reply:
조아요 :+1: 일관되게만 쓰는게 제일 중요하다고 생각합니다 :smile: